docs/src/pages/docs/crocks/Either.md

```diff
@@ -0,0 +1,1346 @@
+instance, wrapping the result of the function. The second function is used when
+`coalesce` is invoked on a `Right` and is used to map the original value,
+returning a new `Right` instance wrapping the result of the second function.
+
+```javascript
+import Either from 'crocks/Either'
+
+import assoc from 'crocks/helpers/assoc'
+import coalesce from 'crocks/pointfree/coalesce'
+import compose from 'crocks/helpers/compose'
+import identity from 'crocks/combinators/identity'
+import hasProp from 'crocks/predicates/hasProp'
+import ifElse from 'crocks/logic/ifElse'
+import map from 'crocks/pointfree/map'
+import mapProps from 'crocks/helpers/mapProps'
+
+const { Left, Right } = Either
+
+// inc :: Number -> Number -> Number
+const inc =
+  x => x + 1
+
+Right(45)
+  .coalesce(identity, inc)
+//=> Right 46
+
+Left(45)
+  .coalesce(identity, inc)
+//=> Right 45
+
+// Record :: { value: Number }
+// hasValue :: Object -> Either Object Record
+const hasValue =
+  ifElse(hasProp('value'), Right, Left)
+
+hasValue({ a: 45 })
+// Left { a: 45 }
+
+hasValue({ value: 45 })
+// Right { value: 45 }
+
+// defaultValue :: Either Object Record -> Either Object Record
+const defaultValue =
+  coalesce(assoc('value', 0), identity)
+
+// incValue :: Either Object Record -> Either c Record
+const incValue = compose(
+  map(mapProps({ value: inc })),
+  defaultValue,
+  hasValue
+)
+
+incValue({ value: 45 })
+
+//=> Right { value: 46 }
+
+incValue({ a: 44 })
+//=> Right { a: 44, value: 1 }
+```
+
+#### swap
+
+```haskell
+Either c a ~> ((c -> d), (a -> b)) -> Either b d
+```
+
+Used to map the value of an `Either` instance and transform a `Left` into a
+`Right` or a `Right` into a `Left`, `swap` takes two functions as its arguments.
+The first function is used to map and transform a `Left` into a `Right`,
+while the second maps and transforms a `Right` into a `Left`. If no mapping of
+the contained values is required for either instance,
+then [`identity`][identity] functions can be used in one or both arguments.
+
+```javascript
+import Either from 'crocks/Either'
+
+import identity from 'crocks/combinators/identity'
+import swap from 'crocks/pointfree/swap'
+
+const { Left, Right } = Either
+
+// swapTypes :: Either c a -> Either a c
+const swapTypes =
+  swap(identity, identity)
+
+swapTypes(Left(45))
+//=> Right 45
+
+swapTypes(Right(23))
+//=> Left 23
+
+// toString :: Number -> String
+const toString =
+  x => x.toString()
+
+// toNumber :: String -> Number
+const toNumber =
+  x => parseInt(x)
+
+// swapValues :: Either Number String -> Either Number String
+const swapValues =
+  swap(toString, toNumber)
+
+swapValues(Left(23))
+//=> Right "23"
+
+swapValues(Right('23'))
+//=> Left 23
+```
+
+#### either
+
+```haskell
+Either c a ~> ((c -> b), (a -> b)) -> b
+```
+
+Used as a means to map and extract a value from an `Either` based on the
+context. `either` takes two functions as its arguments. The first will map any
```

Comment:
comma instead of point after `context` maybe? Just to maintain the same style of the other ones.